build/.night-build.yml

```diff
@@ -8,14 +8,15 @@ pr: none
 # no CI builds
 trigger: none
 
-# scheduled trigger, runs at UTC 8:00 every day which is midnight of GMT-8 
+# scheduled trigger, runs at UTC 8:00 every day which is midnight of GMT-8
 schedules:
 - cron: "0 8 * * *"
   displayName: Nightly Build at midnight
   branches:
     include:
     - master
-    - releases/1.0
+    - main
```

Comment:
Can we remove `master`?

Reply:
I will, but the [steps](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/M2MRenaming/Master-to-Main-renaming-guide.md) explicitly say not to remove master until almost the end. Just to make sure that pipelines keep triggering correctly until the process is done. So I will create another PR to remove master when I get to that point.

Reply:
Oh, I see it is still named `master` in the repo. I thought we had renamed it already.